tools/Crank/BenchmarkApps/DotNet/HelloHttpNet9NoProxy/Hello.cs

```diff
@@ -0,0 +1,29 @@
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+using System.Net;
+
+namespace HelloHttpNet9
+{
+    public class Hello
```

Comment:
Consider using primary ctors for new work:

```suggestion
    public class Hello(ILogger<Hello> logger)
```

Then you don't even need a field for it: just access `logger` throughout your methods.

https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/primary-constructors

Reply:
Switched to primary constructor.